dashboard/config/linux/bits/linux-next.yml

```diff
@@ -3,9 +3,14 @@
 
 kernel:
  repo: git://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next-history.git
- tag: next-20231020
+ tag: next-20231108
 
 config:
   - BCACHEFS_FS
   - BCACHEFS_QUOTA
   - BCACHEFS_DEBUG
+# Prevent writing to mounted devices - otherwise syzbot triggers many
+# inactionable filesystem reports. There have been many discussions
+# on mailing lists re. this, e.g.
+# https://lore.kernel.org/all/0000000000007720b405ff59d161@google.com/T/
+  - BLK_DEV_WRITE_MOUNTED: n
```

Comment:
It's better to put it in the final destination where it will be when it reaches other trees, probably base.yml and predicate with `linux-next`.
I think we also want this enabled on `unmaintained` in future.

Reply:
I did not notice when the BCACHEFS appeared. It already does it differently.
Everything that's linux-next specific should be moved elsewhere when it reaches other trees.
How do we want to handle?
We use the old way too:
```
dashboard/config/linux/bits/base.yml: - NMI_CHECK_CPU: [linux-next]
dashboard/config/linux/bits/bluetooth.yml: - BT_LE_L2CAP_ECRED: [linux-next]
dashboard/config/linux/bits/bluetooth.yml: - BT_HCIBTUSB_POLL_SYNC: [linux-next]
dashboard/config/linux/bits/subsystems.yml: - ADDRESS_MASKING: [x86_64, linux-next]
dashboard/config/linux/bits/subsystems.yml: - X86_USER_SHADOW_STACK: [x86_64, linux-next]
dashboard/config/linux/bits/subsystems.yml: - IOMMUFD_TEST: [linux-next]
dashboard/config/linux/bits/subsystems.yml: - RUNTIME_TESTING_MENU: [linux-next]
```

Reply:
I've refactored the configs.
PTAL.